### PR TITLE
Add DescribeAddresses to lambda-role-trust-policy.json

### PR DESCRIPTION
**Pull request by kwent**
Otherwise lambda function is failing to count EIP.

### CODE REVIEW

File: lambda-role-policy.json [hunk 1]
```diff
@@ -8,10 +8,12 @@
     },
     {
       "Effect": "Allow",
+      "Action": "ec2:DescribeAddresses",
+      "Resource": "*"
+    },
+    {
       "Action": "cloudwatch:PutMetricData",
       "Resource": "*"
```
Comment (MarkRoddy) [inline lambda-role-policy.json]: Am I reading the diff wrong, or is `"Effect": "Allow"` missing for the `cloudwatch:PutMetricData` stanza?

**Comment by kwent** [comment]
You don't read wrong.

It was already missing so I did not add it in this PR.

**Comment by MarkRoddy** [comment]
It looks like it is there on master:
https://github.com/MarkRoddy/count-ec2-instances/blob/master/lambda-role-policy.json#L10

**Comment by kwent** [comment]
@MarkRoddy My bad. I believe it's fixed now.

**Comment by MarkRoddy** [comment]
Thanks!

**Pull request merged**